### PR TITLE
data: EslemOuederni

### PR DESCRIPTION
**Pull request by EslemOuederni**
Added my profile ! 

If there's any edits for me to make please let me know

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5303"></a>

### CODE REVIEW

File: data/EslemOuederni.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+	"name": "Eslem Ouederni",
+	"type": "personal",
+	"bio": "Fullstack developer, I'm looking forward to starting a new journey with open source",
+	"avatar": "https://avatars.githubusercontent.com/u/65737942?v=4",
```
Comment (Panquesito7) [inline data/EslemOuederni.json:5]: The `avatar` field isn't required. Your GitHub avatar will be automatically fetched and updated.
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/EslemOuederni.json (hunk 1 above)
Comment (EslemOuederni) [reply data/EslemOuederni.json:5]: Thank you for pointing that out!

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SurajPratap10 — APPROVED** [review]
LGTM

**Review by rohitverse — APPROVED** [review]

**Review by stevenovak123 — APPROVED** [review]

File: data/EslemOuederni.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+	"name": "Eslem Ouederni",
+	"type": "personal",
+	"bio": "Fullstack developer, I'm looking forward to starting a new journey with open source",
```
Comment (shivam-sharma7) [inline data/EslemOuederni.json:4]:
```suggestion
	"bio": "Fullstack developer, I'm looking forward to start a new journey with open source",
```

**Review by shivam-sharma7 — CHANGES_REQUESTED** [review]

**Review by shivam-sharma7 — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**